docker/falco/Dockerfile

```diff
@@ -1,10 +1,13 @@
 FROM cgr.dev/chainguard/wolfi-base
 
+ARG FALCO_COMMIT_SHA
```

Comment:
Can you add the same `ARG` to other images too? Ie:
* falco-debian
* driver-loader
* driver-loader-buster

Thanks!

Reply:
Done.